arch/risc-v/src/mpfs/mpfs_corespi.c

```diff
@@ -720,6 +731,52 @@ static int mpfs_spi_sem_waitdone(struct mpfs_spi_priv_s *priv)
   return nxsem_tickwait_uninterruptible(&priv->sem_isr, USEC2TICK(timeout));
 }
 
+/****************************************************************************
+ * Name: mpfs_spi_write_tx8
+ *
+ * Description:
+ *   Fill up the TX fifo with one word in configured size frames
+ *
+ * Input Parameters:
+ *   priv     - SPI private state data
+ *   nframes  - Number of 8-bit data per word
+ *   last     - true if this is the last word of the transfer
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+static inline void mpfs_spi_write_tx8(struct mpfs_spi_priv_s *priv,
+                                      unsigned nframes,
+                                      bool last)
+{
+  uint8_t *data8 = (uint8_t *)priv->txbuf;
+  unsigned pos = priv->tx_pos++;
```

Comment:
What is this ++ supposed to do ? Aren't you supposed to move tx_pos forward by nframes * frame_size ?